test/decl/func/vararg.swift

```diff
@@ -27,7 +27,35 @@ func invalidVariadic(_ e: NonExistentType) { // expected-error {{cannot find typ
   { (e: ExtraCrispy...) in }() // expected-error {{cannot find type 'ExtraCrispy' in scope}}
 }
 
-func twoVariadics(_ a: Int..., b: Int...) { } // expected-error{{only a single variadic parameter '...' is permitted}} {{38-41=}}
+func twoVariadics(_ a: Int..., b: Int...) { }
+func unlabeledFollowingVariadic(_ a: Int..., _ b: Int) { } // expected-error {{a parameter following a variadic parameter requires a label}}
+func unlabeledVariadicFollowingVariadic(_ a: Int..., _ b: Int...) { } // expected-error {{a parameter following a variadic parameter requires a label}}
+func unlabeledFollowingTwoVariadics(_ a: Int..., b: Int..., _ c: Int) { } // expected-error {{a parameter following a variadic parameter requires a label}}
+func splitVariadics(_ a: Int..., b: Int, _ c: String...) { }
+func splitByDefaultArgVariadics(_ a: Int..., b: Int = 0, _ c: String...) { }
+
+struct HasSubscripts {
+  subscript(a: Int...) -> Void { () }
+  subscript(a: Int..., b b: Int...) -> Void { () }
+  subscript(a: Int..., b: Int...) -> Void { () } // expected-error {{a parameter following a variadic parameter requires a label}}
+  subscript(a: Int..., b: Int) -> Void { () } // expected-error {{a parameter following a variadic parameter requires a label}}
+  subscript(a: Int..., b b: Int..., c c: Int) -> Void { () }
+  subscript(a: Int..., b b: Int..., c: Int) -> Void { () } // expected-error {{a parameter following a variadic parameter requires a label}}
+  subscript(a: Int..., c c: Int = 0, b: Int...) -> Void { () }
```

Comment:
Test case for `subscript(a: Int..., b: String = "hello, world!") -> Bool { false }` which is in the proposal?

Reply:
👍 , added